interview-questions.md

```diff
@@ -7,51 +7,70 @@ First, without external resources. Challenge yourself to answer from memory as i
 Then, research the question to expand on your answer. Even if you feel you have answered the question completely, there is always something more to learn. Write your researched answer in your OWN WORDS.
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
-
+test
 Your answer:
 
 Researched answer:
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: 
+//Git is used for moving, storing, updating version and sharing files. 
+
+//Git hub is a site that connects to the user(s) local computer allowing collaboration and another place to store code     
 
 Researched answer:
+//Git is a version control system, 
+
+//GitHub is a web-based platform for hosting Git repositories and collaborating on code
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: 
+//logical
 
 Researched answer:
+//Equality, Logical, Comparison
 
 1. What is the difference between an array's index and its value?
 
 Your answer:
+//an arrays index is a series of characters, value are the positions they're in 
 
 Researched answer:
+//The index is the position of an element in the array, while the value is the data stored at that position.
+
 
-4. Compare and contrast arrays and strings. Please include ways in which they are the same and ways in which they are different.
+4. Compare and contrast arrays and strings. Please include ways in which they are the same and ways in which they are 
 
 Your answer:
+//Both arrays and strings are ordered collections of data and hold length properties
+//Arrays can store any data type, while strings can only hold orders of characters 
 
 Researched answer:
+//Arrays and strings are both ordered collections. Arrays can store any type of data and are mutable, while strings store characters only and are immutable
 
 5. Give a brief description of proper pair programming techniques. What are the roles of each person?
 
 Your answer:
+//take turns, take breaks, communicate. 
+//Drivers do all the typing and Navigators plan out the next steps and assist
 
 Researched answer:
+//Pair programming involves two roles: the driver, who writes the code, and the navigator, who observes, reviews, and guides the driver. They collaborate closely, switching roles frequently to enhance productivity, code quality, and learning.
```

Comment:
Collaborating closely and maintaining that communication is key to proper pair programming.